upstream/CVE-2020-8566.json

```
@@ -0,0 +1,56 @@
+{
+  "id": "CVE-2020-8566",
+  "modified": "2020-10-15T22:07:53Z",
+  "published": "2020-10-15T22:07:53Z",
+  "summary": "Ceph RBD adminSecrets exposed in logs when loglevel \u003e= 4",
+  "details": "In Kubernetes clusters using Ceph RBD as a storage provisioner, with logging level of at least 4, Ceph RBD admin secrets can be written to logs. This occurs in kube-controller-manager's logs during provisioning of Ceph RBD persistent claims. This affects \u003c v1.19.3, \u003c v1.18.10, \u003c v1.17.13.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/controller-manager"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.0/AV:L/AC:H/PR:L/UI:N/S:U/C:H/I:N/A:N"
```

https://nvd.nist.gov/vuln/detail/CVE-2020-8566
```suggestion
          "score": "CVSS:3.1/AV:L/AC:H/PR:L/UI:N/S:U/C:H/I:N/A:N"
```

---

strange it do not match to [kubernetes github](https://github.com/kubernetes/kubernetes/issues/95624)

---

You [said](https://github.com/aquasecurity/vuln-list-k8s/pull/85#discussion_r1343785520) it's better to use CVSS from GitHub.
Let's use a vector from GitHub.